spec/unit/compiler/passes/helpers.js

```diff
@@ -68,7 +68,7 @@ beforeEach(function() {
           this.message = () =>
             "Expected the pass not to report an error "
               + "for grammar " + jasmine.pp(grammar) + ", "
-              + "but it did.";
+              + "but it did with message " + jasmine.pp(e.message);
```

Comment:
Please replace “but it did with message” with more grammatically correct “but it reported one with message”. Also, there was a comma at the end of the message.